R/dw_export_chart.R

```diff
@@ -56,6 +56,7 @@ dw_export_chart <- function(chart_id,
                    height = height[1],
                    plain = trimws(tolower(as.character(plain[1]))),
                    scale = scale[1],
+                   zoom = scale[1],
```

Comment:
Would you mind adding some of your rationale from https://github.com/munichrocker/DatawRappr/issues/71#issuecomment-2313182312 into the function documentation on the `scale` argument at the top of this file?

It's worth mentioning that the `zoom` level for PDFs is also populated via the `scale` argument in `dw_export_chart`.

Once this is added, I will be happy to accept the PR. Thanks for investigating and changing this!